core/src/main/resources/org/elasticsearch/tasks/task-results-index-mapping.json

```diff
@@ -0,0 +1,43 @@
+{
+  "result" : {
+    "dynamic" : "strict",
+    "properties" : {
+      "task" : {
+        "properties": {
+          "action": {
+            "type": "keyword"
+          },
+          "cancellable": {
+            "type": "boolean"
+          },
+          "id": {
+            "type": "long"
+          },
+          "parent_id": {
```

Comment:
We're not enabling parent/child on the index so this might be confusing. It is the right name for it, but it still might be confusing.